test/refda/test_exec.c

```diff
@@ -55,6 +56,16 @@ static refda_agent_t agent;
 /// Sequence of executions
 static cace_ari_list_t exec_log;
 
+static int edd_backing_value = 0;
```

Comment:
The logic here is good and the test case is great to have, but this is currently not thread safe. If this is replaced with an `atomic_int` instead (and change read/write to use appropriate functions) it will be safe.